### ISSUE TITLE
Preserve repositories

### ISSUE THREAD

**Issue by TanvirOnGH** [TanvirOnGH/TanvirOnGH#2]
## Reference

- https://zenodo.org/account/settings/github
- https://archive.org

**Comment by DeiseFreire** [comment]
Yes, I agree with you @TanvirOnGH . Taking advantage of my visit to your profile, I would like to say that your personal Readme is beautiful, congratulations.

> "[!COMMENT] ❤️ 
> _Here's why preserving repositories is great:
> Long-Term Access
> Reproducibility
> Knowledge Preservation
> Citation and Credit
> Historical Context.
> Community Engagement
> Mitigating Data Loss
> Open Science and Open Source Culture"_
